python/nano/dev/build_and_install.sh

```diff
@@ -43,7 +43,7 @@ if [ $framework == "basic" ];
 then
   # when framework is specified to "basic", nano will be installed without
   # any framework like Pytorch and TensorFlow
-  pip install $install_options dist/${whl_name}
+  pip install --force-reinstall $install_options dist/${whl_name}
```

Comment:
Here, if we have used this script to install nano, its version will be 2.1.0.dev0. When we try to use this script to install nano again after doing some changes, the version is still 2.1.0.dev0, so pip will refuse to install it again, because there is already a installed nano with same version. Fix this bug.